src/encounterZone.js

```diff
@@ -0,0 +1,130 @@
+import { ActionManager, Color3, ExecuteCodeAction, PredicateCondition } from "@babylonjs/core";
```

Comment:
as always it is recommended to load directly from the respected files (just like the next 2 lines). It does work both ways, but tree shaking will be improved.

Reply:
gotta figure out why my auto-import functionality in VSCode doesn't do that for me...